src/components/Inputs/SearchInput.tsx

```diff
@@ -0,0 +1,54 @@
+import SearchIcon from '@/app/assets/icons/search';
+import { cn } from '@/libs/cn';
+
+import Input from './Input';
+
+interface SearchInputProps
+  extends Omit<React.ComponentProps<typeof Input>, 'className' | 'size'> {
+  placeholder?: string;
+  size?: 'sm' | 'lg';
+}
+
+const sizeClasses = {
+  sm: 'w-[21rem] h-[3.8rem]',
+  lg: 'w-[40rem] h-[4.8rem]',
+};

```

Comment:
변하지 않을 상수라면 `SIZE_CLASSES`로 변경하시는걸 추천드립니다!


```suggestion
const SIZE_CLASSES = {
  sm: 'w-[21rem] h-[3.8rem]',
  lg: 'w-[40rem] h-[4.8rem]',
};
```